prow/cmd/deck/jobs.go

```diff
@@ -34,6 +34,9 @@ const (
 
 type Job struct {
 	Repo        string `json:"repo"`
+	Branch      string `json:"branch"`
+	HeadSHA     string `json:"head_sha"`
+	BaseSHA     string `json:"base_sha"`
```

Comment:
Sure. I just went with what is in the webhook payload, but I agree it's not clear.